tests/integration/components/bs-popover-test.js

```diff
@@ -81,12 +80,6 @@ module('Integration | Component | bs-popover', function(hooks) {
 
     let expectedArrowPosition = versionDependent(225, 219);
 
-    if (isFirefox()) {
-      // Popover arrow has a different positioning by 4px on Firefox
-      // https://github.com/twbs/bootstrap/issues/29485
-      expectedArrowPosition = expectedArrowPosition - 4;
-    }
-
```

Comment:
@jelhan with the test setup now, where we don't run Sass but just import the static BS3/4 CSS, this FF "fix" was not needed anymore. No idea why... 🤔

Reply:
It may have been fixed upstream. But actually not sure. I've rerun the [tests against getbootstrap.com](https://github.com/kaliber5/ember-bootstrap/pull/901#issuecomment-537682135). I'm still seeing a 2px difference. It was 4px before.

![popover-chrome-firefox](https://user-images.githubusercontent.com/4965703/71362171-98ac3980-2595-11ea-9a22-2df9f56e25e4.png)